packages/webdriverio/package.json

```diff
@@ -58,7 +58,7 @@
     "ts-node": "^9.1.1",
     "tsc-silent": "^1.2.1",
     "typescript": "^4.3.5",
-    "webdriverio": "^6.12.1"
+    "webdriverio": "^7.12.4"
   },
   "peerDependencies": {
     "webdriverio": "^6.0.0 || ^5.0.0"
```

Comment:
```suggestion
    "webdriverio": "^7.0.0 || ^6.0.0 || ^5.0.0"
```

Reply:
Oh!!!! :/ I swear I made that change. Must have messed it up with an npm install.